.github/workflows/integrationTest.yml

```diff
@@ -256,19 +260,27 @@ jobs:
           key: "cached_pkg_${{ github.sha }}"
           path: go.mod
 
+      - name: Build Binaries
+        if: steps.cached_pkg.outputs.cache-hit != 'true'
+        working-directory: cwa
+        run: make build package-darwin
+
       - name: Copy binary
         if: steps.cached_pkg.outputs.cache-hit != 'true'
+        working-directory: cwa
         run: |
-          aws s3 cp s3://${S3_INTEGRATION_BUCKET}/integration-test/binary/${{ github.sha }} . --recursive
+          echo cw agent version $(cat CWAGENT_VERSION)
```

Comment:
It would be more recommend to only install the sig and only amazon-cloudwatch-agent.pkg since we don't need every binaries and only force the customers to install in one way instead of everything (e.g `amazon-cloudwatch-agent`)

Reply:
For the integration test, we only upload the pkg. We are not uploading the sig for the pkg and never did for the pkg in the integration tests, this can be added in another pr.